src/index.js

```diff
@@ -1,10 +1,42 @@
 const express = require('express');
-const path = require('path');
 
-const app = express()
+const app = express();
+const cors = require('cors');
+const session = require('express-session');
 const port = process.env.SERVER_PORT || 5000;
+const { Pool } = require('pg');
```

Comment:
Nit: Can we move the package require statements to the top with the Express one?

Reply:
Sure thing!